src/modules/python.rs

```diff
@@ -89,8 +89,10 @@ fn format_python_version(python_stdout: &str) -> String {
         "v{}",
         python_stdout
             .trim_start_matches("Python ")
-            .trim_end_matches(":: Anaconda, Inc.")
             .trim()
+            .split_whitespace()
+            .next()
+            .unwrap_or("?")
```

Comment:
Ideally, this shouldn't be more complex than:
```rust
    let version = python_version
        // split into ["Python", v3.8.6", ...]
        .split_whitespace()
        // return "v3.8.6"
        .nth(1)?;
```
but that is tricky to achieve due to inherited surroundings.

Reply:
@vladimyr Done. I modified your patch slightly.

Reply:
> @vladimyr Done. I modified your patch slightly.

Thank you for fixing my _documentation_ errors :+1: